pkgs/top-level/python-packages.nix

```diff
@@ -6648,14 +6648,14 @@ in {
 
   ipfsapi = buildPythonPackage rec {
     name = "ipfsapi-${version}";
-    version = "0.4.5-pre";
+    version = "0.4.2.post1";
```

Comment:
The release tag needs to be pushed to github, but it's already on pypi:
https://pypi.python.org/pypi/ipfsapi/0.4.2.post1

also see https://github.com/ipfs/py-ipfs-api/pull/105